01-tree-specification.bs

```diff
@@ -220,40 +222,32 @@ I.e., a `tree:path` on `(rdfs:label [sh:alternativePath rdfs:comment ] )` will b
 
 Note: A server is allowed to refer the `tree:path` to a property that is not materialized in the current response. For the client, if it also needs those triples, we assume in this spec that the client has another way of retrieving those, or already retrieved them from another source.
 
-## Comparing strings ## {#strings}
-
-String values have three specific type of relations: the `tree:PrefixRelation`, the `tree:SubstringRelation` and the `tree:SuffixRelation`.
-The string comparison happens using the unicode canonical equivalence.
+## Comparing strings, named node and time literals ## {#sparqlComparators}
 
-Issue: We experimented with server-chosen locales such that `ça suffit` can also be found when following a `tree:PrefixRelation` with a `tree:value "c"` (which at this moment is not supported). That would require an understanding of locales, and [browser/JavaScript support for locales is too low to be useful at this point](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#Locale_identification_and_negotiation).
+When using comparator relations such as `tree:GreaterThanRelation` and `tree:PrefixRelation` the values MUST be compared as defined in the [SPARQL algebra functions](https://www.w3.org/TR/sparql11-query/#expressions).
+See the [`tree:Relation`](#Relation) section.
 
-Also the comparator relations such as `tree:GreaterThanRelation` can be used.
-The strings MUST then be compared using case sensitive unicode ordering.
+### Extention and Recommandation
 
-When a language is set on the `tree:value`, the relation only refers to these language strings. If no language is indicated, it refers to all (including those without).
-
-Particular to the `tree:SubstringRelation`, is that multiple `tree:value` properties can be set. This means the members properties will contain all of the given substrings.
+#### Comparing time literals #### {#time}
+Implementation MUST support `xsd:dateTimeStamp`.
```

Comment:
From the call of 2025-04-02: Let’s remove this in order to be fully in-line with SPARQL